bin/dar

```diff
@@ -41,74 +44,75 @@ my $MINIMUM_IMAGE_VERSION = 1;
+}
+
+my $MENU = <<~'END';
+dar: the cyrus-docker dev tool
+
+Run "dar COMMAND".  Here are some commands:
+ • pull     - pull the latest docker container image
+ • start    - start a container to build in the current dir
+ • prune    - stop and destroy the container for this dir
+ • help     - get more information about how to use dar
+ • run      - run the given command in the container
+ • cyd      - a shortcut for "dar run cyd ..."
+
+And these commands all run "dar cyd CMD" in the container:
+ • build    - configure and compile Cyrus
+ • clean    - "make clean" the repo
+ • makedocs - build the html version of the docs
+ • shell    - run a shell
+ • smoke    - sugar for "build then test"
+ • test     - run the Cassandane test suite
+END
+
+$MENU =~ s{([/|\\]+)}{colored(['bright_blue'], "$1")}ge;
+$MENU =~ s{• \K([^-]+)}{colored(['bright_yellow'], "$1")}ge;
+
 my $HELP = <<'END';
-dar is a little program to help you use the Cyrus docker image for testing your
-work in your own checkout of the cyrusimapd repository.
-
-The idea is that it runs a do-nothing container that's ready to build and test
-Cyrus, and then you can tell it to build and test Cyrus at your whim.  It will
-only manage one container per checkout (as identified by absolute path).
-
-The simplest thing to do is to run "dar smoke", which will start a container
-(if needed), then configure, build, and test Cyrus using the "cyd smoke"
-command packed inside the cyrus-docker image.  If you want to pass arguments to
-smoke, you can run "dar smoke -- test1 test2 test3", for example.
-
-You can run other cyd commands through dar.  If you want to get a shell and
-look into what went wrong on your last test run, you can run "dar shell".  If
-you want to run something other than a cyd command, use "dar run -- XYZ ABC" to
-run XYZ ABC directly in the container.
-
-dar will start a container just in time to run whatever command you want, and
-will leave it running for more commands.  If you want to clean up the container
-it's got running, run "dar --prune".
-
-If you want to start a container before running any particular commands, you
-can run "dar --start", which is mostly useful to allow you to pass "--image" to
-specify an alternate image to use for the container.
-
-The default image is ghcr.io/cyrusimap/cyrus-docker:nightly but you can set a
-different default for all your uses of "dar" by creating the file
-~/.cyrus-docker/config, which should contain a JSON object.  The only
-meaningful key, for now, is "default_image", which provides an alternate
-default image.
+dar helps manage Docker containers for building and testing cyrus-imapd, using
+a copy of the source stored on your computer.
+
+To get a contaner running…
+
+1. cd to your local clone of cyrus-imapd
+2. run "dar pull" to make sure you have the latest cyrus-docker image
+3. run "dar start" to get a running container
+
+Once you have that, you run "dar XYZ" to run "cyd XYZ" inside the container.
```

Comment:
This is no longer accurate, `dar foo` will run `cyd foo` if foo is known command, otherwise it'll return an error. Should probably update this to mention this and also `dar run ...`